src/UI/Component/Input/Field/Date.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/* Copyright (c) 2018 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Input\Field;
+
+/**
+ * This describes the date-field.
+ */
+interface Date extends Input {
```

Comment:
Is it not DateTime? To me Date only excludes time (which is part of the conrol).